rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeRequest.groovy

```diff
@@ -55,6 +55,7 @@ class BakeRequest {
   Boolean enhanced_networking
   String ami_name
   String ami_suffix
+  Boolean upgrade=false
```

Comment:
Can drop the `=false` here and guard the parameterMap instead. This will be more consistent with the way other parameters are handled.